src/java.net.http/share/classes/java/net/http/HttpRequest.java

```diff
@@ -325,6 +335,22 @@ public static HttpRequest.Builder newBuilder(URI uri) {
         return new HttpRequestBuilderImpl(uri);
     }
 
+    /**
+     * Convenience method which returns a GET HttpRequest for the given URI string.
+     *
+     * @param uristring the URI string to get
+     * @return a HttpRequest
+     * @throws IllegalArgumentException if the URI scheme is not supported
+     *         or is otherwise invalid.
```

Comment:
Did you mean "or if the URI string is otherwise invalid?"
Also we could add an `@spec` here to say the returned value is equivalent to 
`HttpRequest.newBuilder(URI.create(uristring).GET().build()`

Reply:
> Did you mean "or if the URI string is otherwise invalid?" 

Yes, that is clearer.

> Also we could add an `@spec` here to say the returned value is equivalent to `HttpRequest.newBuilder(URI.create(uristring).GET().build()`

Not sure that really adds much value imo.